ssm/emissions.py

```diff
@@ -53,6 +53,14 @@ def smooth(self, expected_states, variational_mean, data, input=None, mask=None,
         """
         raise NotImplementedError
 
+    def hessian_log_emissions_prob(self, data, input, mask, tag, x):
+        assert single_subspace, "Only works with a single emission model"
+        # Return (T, D, D) array of blocks for the diagonal of the Hessian
+        T, D = data.shape
+        obj = lambda datat, xt: self.log_likelihoods(datat, input, mask, tag, xt)
+        hess = hessian(obj)
```

Comment:
Note that `hessian` will take the second derivative wrt the first argument.  You want that to be `xt`, not `datat`.

Reply:
Thanks for catching, I fixed this.